src/sentry/static/sentry/app/api.tsx

```diff
@@ -436,12 +436,8 @@ export class Client {
         let responseJSON: any;
```

Comment:
`responseClone` above can be deleted.

Reply:
Ah, sorry. I meant to update this but this should be using `await response.json()` since that may be a little faster than `JSON.parse`.